make/scripts/generate-symbol-data.sh

```diff
@@ -63,7 +63,7 @@ if [ ! -f symbols ] ; then
     exit 1
 fi;
 
-if [ "`hg status .`x" != "x" ] ; then
+if [ "`git status -s .`x" != "x" ] ; then
```

Comment:
Just a suggestion: perhaps you want to use the porcelain format here instead of short? https://git-scm.com/docs/git-status#Documentation/git-status.txt---porcelainltversiongt

Reply:
I agree with @adityamandaleeka, please use `git status --porcelain=v1`. See the git status [documentation](https://git-scm.com/docs/git-status#_porcelain_format_version_1) for the specification for the output.

Reply:
@adityamandaleeka and @edvbld, thanks for the suggestion; I'll update the patch accordingly.